OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -17,7 +17,7 @@ namespace OpenRA.Platforms.Default
 {
 	class Sdl2Input
 	{
-		MouseButton lastButtonBits = (MouseButton)0;
+		MouseButton lastButtonBits = MouseButton.None;
```

Comment:
This was initially simplified to `0` - I have instead tweaked it to the equivalent `MouseButton.None`.